### ISSUE TITLE
Softmax doesn't work with tensorflow 1.4.1

### ISSUE THREAD

**Issue by SidJain1412** [keras-team/keras#10473]
My GPU only supports CUDA 8 (Nvidia GeForce 940M)
CUDA 8 needs CUDNN 7 and tensorflow 1.4.1
Tensorflow 1.4.1 does not support the softmax function.

Is there a workaround or maybe an alternative I can use to the softmax function, because there seems to be no other solution (except for getting a new GPU which supports CUDA 9)

**Comment by brge17** [comment]
keras-team/keras#10440

"As a general rule, the latest release of Keras is compatible with the latest release of TensorFlow as well as the previous release of TensorFlow (in most cases, it is actually compatible with several prior TF releases, but that is not guaranteed).

TF 1.3 is now 5 releases behind (soon to be 6, since 1.9rc is already out), so you should either use a prior version of Keras or use an updated version of TensorFlow."

My guess is your Keras version is new and TensorFlow version is old.

**Comment by tomastheod-ITI** [comment]
@SidJain1412 What do you mean by "Softmax doesn't work with tensorflow 1.4.1"? Can you be more specific?

**Comment by SidJain1412** [comment]
@tomastheod-ITI Got the solution, forgot to close this. 
I had to edit keras code to remove a part that was stopping my program, softmax needed two arguments or something, which was incompatible with my version of tensorflow.
Thank you
